Objects/complexobject.c

```diff
@@ -970,9 +1030,8 @@ complex_new_impl(PyTypeObject *type, PyObject *r, PyObject *i)
         (nbr->nb_float == NULL && nbr->nb_index == NULL && !PyComplex_Check(r)))
```

Comment:
Hmm, it seems there is a coverage regression, not all branches are tested.

Reply:
This should be covered by new tests added in test added in #119635.

Reply:
Now it's better, yet one branch seems to be missed:
```
   1029         [ +  - ]:       3869 :     if (nbr == NULL ||
```
Looks like it's ``nbr!=NULL``.